Makefile

```diff
@@ -3,7 +3,7 @@
 PACKAGES = $(shell go list ./... | grep -v /vendor/)
 
 ifneq ($(shell uname), Darwin)
-	EXTLDFLAGS = -extldflags "-static" $(null)
+	EXTLDFLAGS = -extldflags "-static-libgcc -static-libstdc++" $(null)
 else
```

Comment:
or maybe try building on a different host environment than ubuntu? for local developement you can just run `go build`. There is no need to use the make file and statically compile. For building the distribution we can choose whatever build environment we want via dockerfile